src/blockchain/shelley/utils.js

```diff
@@ -139,7 +141,8 @@ const fragmentToObj = (fragment: any, extraData: {}): TxType => {
         break
```

Comment:
If we have combined the delegation certificate types together: https://github.com/Emurgo/tangata-manu/pull/99/files#r350673514

Here the handling for the `wasm.CertificateType.OwnerStakeDelegation` should also be changed. The `type` in the resulting object should be `StakeDelegation` and additional field `isOwnerStake: true` added